### DISCUSSION TITLE
Recommended for production use?

### DISCUSSION THREAD

**Discussion by visa2learn** [aws/aws-advanced-jdbc-wrapper#268] [Q&A] [answered]
Hi,

I am going through the documentation for this. I think this project got moved from an earlier repo "aws-postgresql-jdbc" to the new one (current). I just happened to see a note in the previous repo

https://github.com/awslabs/aws-postgresql-jdbc
> IMPORTANT Because this project is in preview, we encourage you to experiment with the PostgreSQL driver but DO NOT recommend adopting it for production use

My question is whether aws-advanced-jdbc-wrapper is recommended to be used in production?

Apologies if I have misunderstood this completely.

Thanks

**Comment by davecramer** [comment, marked as answer]
@visa2learn This project has been released for production. We will be supporting this project moving forward. As with any new library you should test it with your application. Let us know if you have any issues.

**Reply by gazal-k** [reply]
The readme currently says "Support for the MySQL JDBC Driver and the MariaDB JDBC Driver is being evaluated.". Would you say it's production ready just to wrap Postgres JDBC driver?

And are there some milestone tests to complete for MySQL or MariaDB driver?
